ee-9/source-transform/ee/src/main/java/org/jboss/as/ee/concurrent/ContextServiceImpl.java

```diff
@@ -22,33 +22,34 @@
 
 package org.jboss.as.ee.concurrent;
 
-import static org.jboss.as.ee.logging.EeLogger.ROOT_LOGGER;
-import static org.wildfly.common.Assert.checkArrayBounds;
-import static org.wildfly.common.Assert.checkNotNullParam;
+import org.glassfish.enterprise.concurrent.spi.ContextSetupProvider;
+import org.wildfly.security.manager.WildFlySecurityManager;
 
 import java.lang.reflect.Proxy;
 import java.security.AccessController;
 import java.security.PrivilegedAction;
 import java.util.Map;
 
-import org.glassfish.enterprise.concurrent.spi.ContextSetupProvider;
-import org.glassfish.enterprise.concurrent.spi.TransactionSetupProvider;
-import org.wildfly.security.manager.WildFlySecurityManager;
+import static org.jboss.as.ee.logging.EeLogger.ROOT_LOGGER;
+import static org.wildfly.common.Assert.checkArrayBounds;
+import static org.wildfly.common.Assert.checkNotNullParam;
```

Comment:
I'm not asking you to fix this, but the old order is correct per our codestyle:

import static

import java
import javax

import other